### ISSUE TITLE
Word Count Test Machine Example

### ISSUE THREAD

**Issue by creese** [FundingCircle/jackdaw#114]
Create a test machine test for the Word Count example.

**Comment by cddr** [comment]
Fixed by #120

**Comment by matthias-margush** [comment]
This happens consistently for me with the `word-count` example:
- Open `word_count_test.clj`
- `m-x cider-jack-in`
- `m-x cider-load-buffer`
- `m-x cider-test-run-ns-tests` works (first time)
- `m-x cider-test-run-ns-tests` fails - every time after the first

```
Error in test-word-count-example
Uncaught exception, not in assertion
   error: org.apache.kafka.streams.errors.ProcessorStateException: Error opening store KSTREAM-AGGREGATE-STATE-STORE-0000000004 at location /tmp/kafka-streams/word-count/0_0/rocksdb/KSTREAM-AGGREGATE-STATE-STORE-0000000004
```
